peps/pep-0612.rst

```diff
@@ -4,14 +4,15 @@ Author: Mark Mendoza <mendoza.mark.a@gmail.com>
 Sponsor: Guido van Rossum <guido@python.org>
 BDFL-Delegate: Guido van Rossum <guido@python.org>
 Discussions-To: typing-sig@python.org
-Status: Accepted
+Status: Final
 Type: Standards Track
 Topic: Typing
-Content-Type: text/x-rst
 Created: 18-Dec-2019
 Python-Version: 3.10
 Post-History: 18-Dec-2019, 13-Jul-2020
 
+.. canonical-doc:: :external+python:class:`typing.ParamSpec` and
```

Comment:
I would like a canonical doc link to the typing spec instead. ParamSpec is a very subtle part of the type system and there is a lot more to it than described by the CPython runtime docs.

Reply:
To this page? 

https://typing.readthedocs.io/en/latest/spec/index.html

Or this?

https://typing.readthedocs.io/en/latest/spec/generics.html#paramspec